app/src/interfaces/presentation-links/index.ts

```diff
@@ -79,6 +98,40 @@ export default defineInterface({
 									font: 'monospace',
 									placeholder: 'https://example.com/articles/{{ id }}/{{ slug }}',
 								},
+								conditions: [
+									{
+										rule: {
+											actionType: {
+												_eq: 'flow',
+											},
+										},
+										hidden: true,
+									},
+								],
+							},
+						},
+						{
+							field: 'flow',
+							type: 'string',
+							name: '$t:flow',
+							meta: {
+								width: 'full',
+								interface: 'system-flow-select',
+								hidden: true,
+								note: '$t:interfaces.presentation-links.select_flow_note',
+								options: {
+									collectionName: collection,
+								},
+								conditions: [
+									{
+										rule: {
+											actionType: {
+												_eq: 'flow',
+											},
+										},
+										hidden: false,
+									},
+								],
```

Comment:
Now that I have a much better understanding of all this. It might be worth just amending the passed `flow` to pass the whole or partial flow object instead of just the Id.

That way we won't have to mess around with passing `isActiveFlow` functions and extrapolating context from useFlows and can just access that logic directly. 🤔 

We'd end up with the following structure for the Link type

```
type Link = {
	icon: string;
	label: string;
	type: string;
	actionType: 'url' | 'flow';
	url?: string;
	flow?: Partial<FlowRaw>; <----- Something like this
};
```

Or is there something I'm not thinking of?